pkgs/applications/editors/vim/plugins/aliases.nix

```diff
@@ -95,6 +95,7 @@ mapAliases (with prev; {
   neosnippet          = neosnippet-vim;
   The_NERD_Commenter  = nerdcommenter;
   The_NERD_tree       = nerdtree;
+  nvchad-extensions   = throw "nvchad-extensions: plugin has been deleted by upstream"; # Added 2023-08-19
```

Comment:
this will get overwritten by the deprecation, so should be safe to remove, `vimPlugins: resolve github repository redirects` can be squashed (rebased) into this commit as well since it is just sorting